theano/tensor/nnet/abstract_conv.py

```diff
@@ -308,6 +308,17 @@ def perform(self, node, inp, out_):
             'Did you exclude both "conv_dnn" and "conv_gemm" from '
             'the optimizer? Is cudnn available and does the GPU support it?')
 
+    def R_op(self, inputs, eval_points):
+        rval = None
+        if eval_points[0] is not None:
+            rval = self.make_node(eval_points[0], inputs[1]).outputs[0]
+        if eval_points[1] is not None:
+            if rval is None:
+                rval = self.make_node(inputs[0], eval_points[1]).outputs[0]
+            else:
+                rval += self.make_node(inputs[0], eval_points[1]).outputs[0]
+        return [rval]
```

Comment:
There has been some confusion regarding whether the output should be `None` if *either* `eval_point` is missing, or if *both* are missing.
Since this is the same as `ConvOp`'s implementation, I think it is good for now.